dj/api/nodes.py

```diff
@@ -207,64 +208,6 @@ def read_node(
     return node  # type: ignore
 
 
-def check_databases_registered(
-    tables: List[CreateTable],
-    session: Session,
-) -> Dict[str, Database]:
-    """
-    Verify that the referenced databases are registered in the system.
-    """
-
-    database_names = {table.database_name for table in tables}
-    query = select(Database).where(
-        # pylint: disable=no-member
-        Database.name.in_(database_names),  # type: ignore
-    )
-    databases = {db.name: db for db in session.exec(query).all()}
-    missing_databases = database_names - databases.keys()
-    if missing_databases:
-        raise DJException(message=f"Database(s) {missing_databases} not supported.")
-    return databases
-
-
-def create_source_node_revision(
```

Comment:
Since the database check and dimension column additions are moved to the separate API endpoints, this function became just instantiating the `NodeRevision` so I removed this and moved the instantiation to the create node endpoint function.